utilities/ovs-p4ctl.in

```diff
@@ -1,6 +1,6 @@
 #! @PYTHON3@
 #
-# Copyright(c) 2021 Intel Corporation.
+# Copyright(c) 2021, 2022 Intel Corporation.
```

Comment:
In few places, you have Copyright (c) and in few other places, you have Copyright(c). Please be consistent

Reply:
The norm is to have spaces before and after the **(c)**.

Reply:
Fixed